spark/pyspark/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7.3-stretch
+
+ARG BAZEL_REPO="deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8"
```

Comment:
@ValentinMouret I think you don't need ARG, but just ENV here. Because in the makefile i could not find any usage of BAZEL_REPO